SUMMARY.md

```diff
@@ -40,6 +40,7 @@
   - [Deref Polymorphism](./anti_patterns/deref.md)
 
 - [Functional Programming](./functional/index.md)
+  - [Declarative vs Imperative](./functional/paradigms.md)
```

Comment:
```suggestion
  - [Programming paradigms](./functional/paradigms.md)
```

Reply:
Maybe better "Programming paradigms" as the page title?

Reply:
Yeah, would be clearer, I thought that it looks repeated, because over it will be `Funtional Programming` and then the next line starts with `Programming Paradigms` again. Like Programming ... Programming.

Reply:
done in 0e89dacb7